shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -208,7 +211,7 @@
 // A number of methods in this class have the same implementation as FlutterFragmentActivity. These
 // methods are duplicated for readability purposes. Be sure to replicate any change in this class in
 // FlutterFragmentActivity, too.
-public class FlutterActivity extends Activity
+public class FlutterActivity extends ComponentActivity
```

Comment:
Are there any implications to existing third-party subclasses of FlutterActivity from changing this?

Reply:
Turns out this is not needed anymore. Thus also making the lifecycle changes unnecessary.